instrumentation/net/http/otelhttp/handler.go

```diff
@@ -170,13 +178,21 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	h.handler.ServeHTTP(w, r.WithContext(ctx))
 
-	setAfterServeAttributes(span, bw.read, rww.written, rww.statusCode, bw.err, rww.err)
+	if isReqBodyNil {
+		setAfterServeAttributes(span, 0, rww.written, rww.statusCode, nil, rww.err)
+	} else {
+		setAfterServeAttributes(span, bw.read, rww.written, rww.statusCode, bw.err, rww.err)
```

Comment:
Can you avoid these conditionals by setting `bw` to an empty `bodyWrapper` above and not replacing `r.Body` with it?  It looks like these are just providing the empty values for those fields if the request body is nil.

Reply:
Sounds like a good idea. Let me try that.

Reply:
Done.